src/schedule/mod.rs

```diff
@@ -43,26 +43,96 @@
 //! welcome_secret) is represented by its own struct to ensure that the keys are
 //! not confused with one-another and/or that the schedule is not derived
 //! out-of-order.
+//!
+//! ## The real key schedules
+//! The spec isn't great here. The key schedule is actually
```

Comment:
These kind of comments are not very helpful for others. 

From what I gather, the difference between the two key schedules is only the beginning, namely:

```
                  init_secret_[n-1]
                         |                       
                         V                         
    commit_secret -> KDF.Extract                         
                         |
                         V                         
                   DeriveSecret(., "joiner")
                         |
                         V
```

I think it would be much clearer to explain the discrepancy between the spec and the implementation.

Reply:
I'm curious why you think the extract-expand of the commit and init secrets into the joiner secret is not actually part of the key schedule. Can you elaborate on that?

Reply:
Well there are both flows you everyone can just compare to see the differences. Also, I'd expect this to change in the spec before its done. And then we should update the comment here.
I updated the description here to explain a little more.